stetho/src/main/java/com/facebook/stetho/Stetho.java

```diff
@@ -290,7 +294,21 @@ public DefaultInspectorModulesBuilder runtimeRepl(RuntimeReplFactory factory) {
      * {@link Context#getDatabasePath} method will be used by default if not overridden here.
      */
     public DefaultInspectorModulesBuilder databaseFiles(DatabaseFilesProvider provider) {
-      mDatabaseFiles = provider;
+      mDatabaseFilesProvider = provider;
+      return this;
+    }
+
+    /**
+     * Extend and provide additional database drivers. Currently two database drivers are supported
+     * in this lib: <br>
+     *   1. <code>SqliteDatabaseDriver</code> - Presents sqlite databases and all tables of the app.<br>
```

Comment:
nit: instead of `<br>` we should use:

```html
<ol>
<li>...</li>
</ol>
```

This will automatically create a new paragraph block.